packages/ui/src/components/molecules/RegionBar/styles.scss

```
@@ -24,16 +24,22 @@
   // Postal Code
   --fs-region-bar-postal-code-margin-right : auto;
 
+  display: flex;
+
   // --------------------------------------------------------
   // Structural Styles
   // --------------------------------------------------------
 
   width: var(--fs-region-bar-width);
 
```

```suggestion
  // --------------------------------------------------------
  // Structural Styles
  // --------------------------------------------------------
  
  display: flex;
  width: var(--fs-region-bar-width);
```suggestion
  display: flex;

  // --------------------------------------------------------
  // Structural Styles
  // --------------------------------------------------------

  width: var(--fs-region-bar-width);

```
```